.github/workflows/Aqua.yml

```diff
@@ -0,0 +1,20 @@
+name: Aqua
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: '1'
+      - name: Aqua.jl
+        run: |
+          julia --color=yes -e 'using Pkg; Pkg.add("Aqua"); Pkg.develop(path="."); using Aqua, JSOTemplate; Aqua.test_all(JSOTemplate)'
```

Comment:
```suggestion
          julia --color=yes -e 'using Pkg; Pkg.add("Aqua"); Pkg.develop(path="."); using Aqua, JSOTemplate; Aqua.test_all(JSOTemplate)'
```
```suggestion
          julia --color=yes -e 'using Pkg; Pkg.add("Aqua"); Pkg.develop(path="."); using Aqua, JSOTemplate; Aqua.test_all(JSOTemplate, ambiguities=false); Aqua.test_ambiguities(JSOTemplate)'
```

Reply:
It will test ambiguities of dependencies if we run `Aqua.test_all(JSOTemplate)`. It works fine for modules without dependency such as Krylov.jl but I encounter the problem with SuiteSparseMatrixCollection.jl.

Reply:
Thanks for the suggestion, I've committed them.